plugins/cisco_firepower_management_center/icon_cisco_firepower_management_center/actions/block_url_policy/action.py

```diff
@@ -16,62 +15,93 @@ def __init__(self):
             output=BlockUrlPolicyOutput(),
         )
 
-    def make_policy(self, fmc: fmcapi.FMC, policy_name: str):
-        acp = fmcapi.AccessPolicies(fmc=fmc)
-        acp.name = policy_name
-        policy = acp.get()
-        if "paging" in policy:
-            if policy.get("paging", {}).get("count") == 0:
-                policy = acp.post()
-        self.logger.info(policy)
-        return policy
-
-    @staticmethod
-    def _make_url_object(fmc: fmcapi.FMC, name: str, fqdn: str) -> str:
-        url = fmcapi.URLs(fmc=fmc)
-        url.name = name
-        url.url = fqdn
-        url.post()
-        url.get()
-        return url.id
-
-    @staticmethod
-    def _make_rule(fmc: fmcapi.FMC, policy: dict, rule_name: str, urls: dict) -> str:
-        acr = fmcapi.AccessRules(fmc=fmc)
-        acr.name = rule_name
-        acr.urls = urls
-        acr.acp_id = policy.get("id")
-        acr.URL = policy.get("rules", {}).get("links", {}).get("self")
-        acr.action = "BLOCK"
-        acr.enabled = True
-        acr.post()
-        return acr.id
+    def get_policy_object(self, policy_name: str) -> Union[dict, None]:
+        """
+
+        :param policy_name:
+        :return:
+        """
```

Comment:
I know the code is self-explanatory but would be awesome to see a short description in each method 😄

Reply:
Yes I was going to fill these out once I got the PR pushed up 😄